modin/pandas/groupby.py

```diff
@@ -514,6 +514,13 @@ def try_get_str_func(fn):
                 func, **kwargs
             )
             func_dict = {col: try_get_str_func(fn) for col, fn in func_dict.items()}
+            if any(isinstance(fn, list) for fn in func_dict.values()):
+                # multicolumn case
+                # putting functions in a `list` allows to achieve multicolumn in each partition
+                func_dict = {
+                    col: fn if isinstance(fn, list) else [fn]
+                    for col, fn in func_dict.items()
+                }
```

Comment:
It is not very convenient to rewrite it, maybe you will like my other implementation.

Reply:
I wonder if this will work?

```python
-                return fn.__name__ if callable(fn) and fn.__name__ in dir(self) else fn
+                return [fn.__name__] if callable(fn) and fn.__name__ in dir(self) else [fn]
```

Reply:
No, there is a recursive call for functions that are already in the list. In this case we will get a list of lists.

Reply:
Ah, I see, thanks! Let's please return back the first commit you did, I think it looks a bit clearer and cleaner.

Reply:
We could probably change the code to something like this though.

```python
- func_dict = {col: try_get_str_func(fn) for col, fn in func_dict.items()}
+ funcs = dict()
+ for col, fn in func_dict.items():
+     f = try_get_str_func(fn)
+     funcs[col] = f if isinstance(f, list) else [f]
```

I am not sure about performance and cleanliness of this code and your second commit. Do you think what would be better?

Reply:
Given `func_dict` is not a large iterable object, I think it shouldn't add a big performance penalty when looping over it twice. Anyway, I would like to listen to your thoughts about my suggestion above and your first commit.

Reply:
I reverted the changes.
I agree with you about performance.

If try to connect your code and the condition under which it should work, it will turn out to be about the same as it was before I rolled back to my first commit. What do you think?

Reply:
I think we can leave as is now.